foxglove/cmd/utils.go

```diff
@@ -228,3 +229,13 @@ func maybeConvertToRFC3339(timestamp string) (string, error) {
 	}
 	return parsed.Format(time.RFC3339), nil
 }
+
+func TokenIsApiKey(token string) bool {
```

Comment:
Existing installs won't have defined "auth_type", so I don't think we should assume session auth in that case. Otherwise, the user will get an error when trying to fetch /me. I was thinking we could indicate some indeterminate state to handle that, rather than a bool.

But the most practical step might be to define an Unknown AuthType and (if unknown) fall back to using the "fox_sk_" logic you had before. Existing API keys will have that prefix.

@wkalt thoughts?

Reply:
(If we don't want to use that fallback, then the `token` parameter here is unneeded.)

Reply:
my general thought is that the backend should know what kind of key a given key is, and be able to tell the client. It seems really fragile to bake this kind of awareness into the client tool. If we wanted something non-fragile, JWT claims (in the token, i.e revise our token strategy) seems like another way we could do it.